packages/combo-box/src/vaadin-combo-box-data-provider-mixin.js

```diff
@@ -249,7 +247,9 @@ export const ComboBoxDataProviderMixin = (superClass) =>
         this.loading = this.__dataProviderController.isLoading();
       }
 
-      super.requestContentUpdate();
+      if (this._scroller) {
+        this._scroller.requestContentUpdate();
+      }
```

Comment:
It seems this call is also unnecessary. Controller state changes will anyway trigger property updates that will be passed down to the scroller.

Reply:
Done